kolibri/plugins/facility/views.py

```diff
@@ -33,11 +38,15 @@ def download_csv_file(request, filename):
     response["Content-Type"] = "text/csv"
 
     # set the content-disposition as attachment to force download
-    response["Content-Disposition"] = "attachment; filename=users_{}.csv".format(
-        datetime.now().strftime("%Y%m%d_%H%M%S")
+    exported_filename = pgettext(
+        "Name of a CSV file exporting the users of the facility", "users_{}.csv"
```

Comment:
`Default name for the exported CSV file of facility user data. Please keep the underscore between words in the translation.`